docs/defaulting.md

```diff
@@ -0,0 +1,180 @@
+# Defaulting Service Proposal
+
+This Proposal fills a gap between KubeVirt as Runtime and Higher level
+management, like proposed in #272. It lays out a design wich overcomes the
+necessity for asynchronous specification updates, which has it's origin in the
+fact that a lot of the defaulting is done on the host, so after scheduling.
+
+With the proposal here, which will provide a synchronous way of fully
+populating a VM specification, we come much closer to the meaning of a Pod
+specification in Kubernetes, but most important, this allows decoupling
+administrative tasks completely from the runtime, when manipulating VM Specs
+(the need to run VMs, to get a full VM spec).
```

Comment:
The title is good. It actually replaces the defaulting of libvirt on the node level.